notebooks/official/tabular_workflows/prophet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1291 @@
+{
```

Comment:
<p>Avoid future tense. "This tutorial will use..." -&gt; "This tutorial uses..."</p><p> </p><blockquote>"Prophet will only be using historical sales, so it should not be trivial to get decent evaluation metrics."</blockquote><p>"Because Prophet uses historical sales data only, generation of reasonable evaluation metrics is a non-trivial task." Is this an accurate reformulation? Can we make the same statement with respect to ARIMA+?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>